### ISSUE TITLE
Discord, Matrix?

### ISSUE THREAD

**Issue by Nyabinary** [polyzium/project-corrosion#5]
What the title says

**Comment by wallabra** [comment]
What should adding Discord or Matrix support to a music sequencer program entail?

**Comment by Nyabinary** [comment]
More communication with potential contributors and just a general community imo

**Comment by wallabra** [comment]
I think we're in far too early a stage for such community facilities, but let's see what @polyzium thinks. Once there is more of a team, I think we can have a development team Discord or something.

**Comment by polyzium** [comment]
Holy shit, this has been open for two weeks? Sorry guys I haven't been working on the project lately, got too carried away by work and failure...

but yeah, what Gus says, this project is in its very early stages, too young and too underdeveloped to have a community. Besides, this is mainly for me to learn and also perhaps a very niche group of people, since trackers are pretty much a dead breed of sequencers, but I digress.

I was actually thinking about getting a community for this software at some point in time. I initially wanted to link this project with Friction Visions LLC and their community, but unfortunately there has been some drama lately and as a result, it died. Former members have left and made Mendingchord, which I'm a part of.

So in conclusion, I think it's just too early, there are some people in Mendingchord who are following my project, some of them are even Rust devs, so maybe we can stay there for a while before a larger following comes in.
